web/src/components/Upload/Uploads/Uploads.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+
+import type { FindUploads } from 'types/graphql'
+
+import TableBuilder from 'src/components/TableBuilder/TableBuilder'
+
+import { columnDefs } from './columns'
+
+const UploadsList = ({ uploads }: FindUploads) => {
+  // Uploads.validations is an array that can contain both valid and invalid validations,
+  // We store the latest valid and invalid entry
+  function findLatestValidationAndInvalidationDatesFromUpload(upload) {
```

Comment:
This code could be a lot cleaner, ideally we'd want to pass `uploads` straight into `<TableBuilder />` (or maybe we could process `validations` on the backend?).

In ARPA, uploads have 2 date strings for validated and invalidated dates, I tried to replicate that here from the `validations` array by getting the latest possible date value (one for the invalidation date, one for the validation one), does this look right? @as1729

Reply:
Confirmed via Slack that we will use the latest possible date value.